### DISCUSSION TITLE
Server problem

### DISCUSSION THREAD

**Discussion by meferin** [ltsp/ltsp#250] [Q&A] [answered]
I've got problem with server, everything is going fine, screen working. When computer was started show this.
![Uploading 1581673952863920886839547261777.jpg…]()

**Comment by alkisg** [comment]
@meferin, see https://github.com/ltsp/community/issues/101#issue-565218309, you probably submitted the issue before the screenshot had finished uploading.

So now you need to edit it and upload it again for us to see it.

**Comment by meferin** [comment]
![20200214_104040](https://user-images.githubusercontent.com/61047076/74635081-88fc3c00-5165-11ea-8220-8178b8633614.jpg)

**Comment by meferin** [comment]
I'm sorry, i hope you can read something from this image

**Comment by alkisg** [comment]
It sounds like a problem with your PAM config. Which programs did you install that modify the PAM config?
You may also come to the IRC channel for live support. Go to https://ltsp.org and click on the IRC channel link.

**Comment by meferin** [comment]
Could this be a problem with partitioning from the installation level? I did everything according to the LTSP instruction, the Boot partition did not work, but I did. I did nothing with PAM and everything was as it was. And how can i configure PAM?

**Comment by alkisg** [comment]
In general you don't need to configure PAM, it gets automatically configured when you install LDAP or other authentication methods.

If the Debian/Ubuntu installer or some other package failed at some point and it left a broken debconf database, then this might also cause the Debian/Ubuntu PAM scripts to fail.

Which distro/version is that? Let me suggest IRC once more, as this doesn't sound related to LTSP.

**Comment by trakinfo** [comment]
The distro is Ubuntu desktop 18.04

**Comment by alkisg** [comment]
Hi, I just verified that Ubuntu desktop 18.04 works fine here, so it's probably a broken installation issue, not an LTSP issue.

**Comment by jblackbu01** [comment]
I'm having the same issue and I'm using a fresh install of Ubuntu Desktop 18.04. Attempting an LTSP install on Ubuntu Desktop 19.10. More details to follow.

**Comment by meferin** [comment]
After reinstalling and using the default settings the problem disappeared. I don't know what was wrong before.

**Comment by sebastiendu** [comment]
Same issue here. The last lines displayed on the screen are:
<pre>
Use of uninitialized value in concatenation (.) or string at /usr/share/perl5/Debconf/DbDriver/File.pm line 49, <DEBCONF_CONFIG> chunk 3.
Use of uninitialized value in -e at /usr/share/perl5/Debconf/DbDriver/File.pm line 51, <DEBCONF_CONFIG> chunk 3.
Use of uninitialized value in sysopen at /usr/share/perl5/Debconf/DbDriver/File.pm line 54, <DEBCONF_CONFIG> chunk 3.
Use of uninitialized value in concatenation (.) or string at /usr/share/perl5/Debconf/DbDriver/File.pm line 54, <DEBCONF_CONFIG> chunk 3.
debconf: DbDriver "config": could not open
Could not configure PAM for SSH authentication!
LTSP command failed: /usr/share/ltsp/client/login/pamltsp install
Stopping plymouth
Aborting ltsp
</pre>

debconf looks for file /var/cache/debconf/config.dat, which exists on the server but not on the booting client, where /var is empty.

Now why is /var empty on the client ?

**Comment by alkisg** [comment]
> Now why is /var empty on the client ?

You probably use a separate /var partition, and you didn't tell `ltsp image` about it.
Please read this one: https://github.com/ltsp/ltsp/discussions/347

**Reply by sebastiendu** [reply]
I did not see your answer before posting mine, thank you anyway, that worked !
However I could not find the explanation in the current manual page https://ltsp.org/man/ltsp-image/

**Comment by sebastiendu** [comment, marked as answer]
I wrote:
> Now why is /var empty on the client ?

Because on the server, /var is a distinct disk partition from /.
This problem is explained in [issue 43](https://github.com/ltsp/ltsp/issues/43)
Therefore the fix for this is to replace command
<pre>
ltsp image /
</pre>
by
<pre>
ltsp image /,,/var,subdir=var
</pre>
